packages/format-currency/package.json

```
@@ -0,0 +1,27 @@
+{
+	"name": "format-currency",
+	"version": "1.0.0",
+	"description": "JavaScript library for formatting currency",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/wp-calypso.git"
+	},
+	"keywords": [
+		"currency",
+		"internationalization"
+	],
+	"author": "Automattic, Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"dependencies": {
+		"i18n-calypso": "^4.0.0"
```

this should be a `file:` over to the local i18n-calypso package
```suggestion
		"i18n-calypso": "file:../i18n-calypso"
```